<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>game</class>
 <widget class="QMainWindow" name="game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>game</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>681</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/sky.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="scorepic">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>560</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;David&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>470</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Algerian&quot;;</string>
    </property>
    <property name="text">
     <string>SCORE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Algerian&quot;;</string>
    </property>
    <property name="text">
     <string>time(s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;AR DARLING&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/gre_line.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>160</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/red_star.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>320</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/bumb.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="stone4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>80</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;Angsana New&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="stoneL">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>240</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;Angsana New&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="stone5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>390</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;Angsana New&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>450</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/win.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="star">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>550</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 26pt &quot;Algerian&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
